### PR TITLE
JinZhi's edit

### PR DESCRIPTION
**Pull request by jinzhihoi**
videogamegranpa

### CODE REVIEW

**Comment by boonlimk** [comment]
can you change and make sure you use your proper name?

**Comment by jinzhihoi** [comment]
You dont like videogame Grandpa ? ;(

File: testText.txt [hunk 1]
```diff
@@ -1,2 +1,7 @@
 BL - Hello World
 Can i edit this directlu?
+
+
+
+
+Whats up its me , videogamegranpa!
```
Comment (boonlimk) [inline testText.txt]: Please change the name

**Review by boonlimk — APPROVED** [review]
No i dont, thank you for changing it.
Videogame grandma is better

**Pull request merged**